plugins/modules/ipapwpolicy.py

```diff
@@ -325,7 +325,48 @@ def main():
 
     ansible_module.params_fail_used_invalid(invalid, state)
 
-    if gracelimit is not None:
+    # Ensure parameter values are valid and have proper type.
+    def int_param(value, param):
```

Comment:
It would be nice to have a better name for the function as "" is valid also.

Reply:
Changed to `int_or_empty_param`.